### ISSUE TITLE
CommonJS Exports

### ISSUE THREAD

**Issue by finkef** [jaredh159/tailwind-react-native-classnames#87]
Hey, I'm currently writing a babel plugin using babel-plugin-macros for statically extracting and converting the tailwind styles for React Native and React Native Web, including support for SSR with media queries. (Working on some finishing touches, adding docs, etc. before publishing in the coming days 🤞)
Before, I was using tailwind-rn to convert the styles, but had no way of using the `tailwind.config.js` during Babel runtime. Thankfully, I found your amazing v2 rewrite and just migrated to using your lib under the hood. Unfortunately, babel-plugin-macros doesn't support ESM, so I get errors when running metro.

Would be awesome if you could add a CommonJS target to the compilation. Since I'm running a very similar setup, here's how I tackled cjs output in addition to ESM:

---

```js
// tsconfig.cjs.json
{
  "extends": "./tsconfig.json",
  "compilerOptions": {
    "module": "CommonJS",
    "outDir": "./dist/cjs",
    "declaration": false
  }
}
```

```js
// package.json
{
  // ...
  "main": "dist/cjs",
  "module": "dist",
  // ...
  "scripts": {
    "compile": "fldev ts:compile && fldev ts:compile -p tsconfig.cjs.json",
  }
}
```

**Comment by jaredh159** [comment]
hey, thanks for the issue.  yup, i'm up for adding support for this, and I've opened up a PR: #91.  I'm not super familiar with this stuff, wondering if you could maybe review the PR for me, and maybe try it out in your project -- I published the code in the PR as a beta version which you can install with `npm install twrnc@next`.

I tried it out locally in a RN app, and I lost typescript types until i put the `"types":` field in `package.json`, but after that it seemed to be working fine.

I'd like it if someone who was using RN-web could test it as well. maybe @smlarkin or @oste would be willing to check that out.

**Comment by finkef** [comment]
#93 implements the reorganization on top of PR #91 adding CommonJS compilation.
I have a RNW + Expo + Next.js project setup locally that I can test against once we get a new release.

**Comment by finkef** [comment]
Closed by #93.
